### DISCUSSION TITLE
How to map localhost:xxxx

### DISCUSSION THREAD

**Discussion by skolari93** [dokku/dokku#5742] [Q&A]
I have a tool outside of dokku and docker running on localhost:xxxx. How can I use a dokku domain to route a domain to this localhost port? I tried chapter [Routing an app to a known ip:port combination](https://dokku.com/docs/networking/network/#routing-an-app-to-a-known-ip:port-combination) but this didn't work on my ubuntu server.

**Comment by josegonzalez** [comment]
What service is it that is running on the host?

**Comment by skolari93** [comment]
listmonk on localhost:9000

------- Original Message -------
josegonzalez ***@***.***> schrieb am Sonntag, 19. März 2023 um 9:41 vorm.:
